package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@qvalia/knex-aws-data-api",
+  "version": "1.0.0",
+  "description": "Knex plugin that uses AWS Data API internally to execute SQL queries. Postgres & Mysql",
+  "main": "index.js",
+  "files": [
+    "src/**/*.js"
+  ],
+  "scripts": {
+    "lint": "eslint src",
+    "test-integration": "jest --config=jest.config.js --testMatch **/test/**/*.test.js",
+    "test": "jest --config=jest.config.js --testMatch **/src/**/*.test.js",
+    "package": "rm -rf node_modules && npm install --prod"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Qvalia/knex-aws-data-api.git"
+  },
+  "keywords": [
+    "qvalia",
+    "knex",
+    "data",
+    "api",
+    "rds",
+    "aurora",
+    "serverless",
+    "mysql",
+    "postgres"
+  ],
+  "author": "Qvalia",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Qvalia/knex-aws-data-api/issues"
+  },
+  "homepage": "https://github.com/Qvalia/knex-aws-data-api#readme",
+  "peerDependencies": {
+    "knex": "^2.4.2"
+  },
+  "devDependencies": {
+    "rewire": "^6.0.0",
+    "@aws-sdk/client-rds-data": "^3.332.0",
+    "dotenv": "16.0.3",
+    "eslint": "^8.39.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-node": "^4.1.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-jest": "^27.2.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "jest": "^29.5.0",
+    "knex": "^2.4.2",
+    "knex-migrate": "1.7.4",
+    "mysql": "2.18.1",
+    "pg": "8.10.0",
+    "prettier": "^2.8.7"
+  },
+  "dependencies": {
+  }
```

Comment:
Yes, we only need two runtime dependencies `@aws-sdk/client-rds-data` and `knex`. Since the plugin should run in lambdas, AWS sdk is included in a lambda. Added knex as a peer dependency so that npm will make sure knex is installed when this library added.

But you make a good point, I'll add these as comments on the Readme. So that the dependencies are clear when an npm error pops up.